lib/elixir/lib/file.ex

```diff
@@ -442,6 +442,23 @@ defmodule File do
   end
 
   @doc """
+  Rename the `source` file to `destination` file.
+
+  It returns `:ok` in case of success, returns
+  `{:error, reason}` otherwise
+
+  Note: we are following the erlang rename semantics and you are
```

Comment:
We don't need to say they are erlang semantics. Even for cp, which is named after the unix one, we chose to disable this behaviour because it is craziness: https://github.com/aforward/elixir/blob/f/File.rename/lib/elixir/lib/file.ex#L475. We can just have a similar footnote. :)